Source/Diagnostics/MultiDiagnostics.H

```diff
@@ -25,14 +25,24 @@ public:
     void ReadParameters ();
     /** \brief Loop over diags in alldiags and call their InitDiags */
     void InitData ();
-    /** \brief Called at each iteration. Compute diags and flush. */
-    void FilterComputePackFlush (int step, bool force_flush=false);
+    /** \brief Called at each iteration. Compute diags and flush.
+     *
+     * \param[in] step simulation timestep when diagnostics is invoked
+     * \param[in] force_flush     whether to enforce flushing data even if plot interval
+     *                            criterion is not satisfied
+     * \param[in] BackTranform    whether the diagnostics call is for regular or back transformed diagnostics
+     */
+    void FilterComputePackFlush (int step, bool force_flush=false, bool BackTransform=false);
```

Comment:
See below, I would not complicate that interface further - let's instead handle this in the body of the function.